Classes/Hooks/UserFunc.php

```diff
@@ -62,81 +62,4 @@ public function displayThumbnail(&$params)
         }
         return $output;
     }
-
```

Comment:
Is this part related to TYPO3 version?

Reply:
This getDocumentType()-function is not used in TYPO3 9.5 anymore. The protected loadDocument() was only needed by getDocumentType(). That's why I removed it.